crowdsec-docs/unversioned/service_api/introduction.md

```diff
@@ -0,0 +1,39 @@
+---
+id: intro
+title: Introduction
+---
+
+:::info
+
+This feature is only accessible to paying users of Crowdsec.
+
+:::
+
+
+## Purpose
+
+The service API allows you to manipulate your Crowdsec stack without using the console interface directly.
```

Comment:
un peu keuse/, et Service avec un grand S, un peu de bold et tout et tout.

The **Service API** provides CrowdSec's SaaS features via various endpoints.
The current endpoints allow you to : ...